What's New: Correlations in 10.2 (#76505)
grafana/grafana: docs/sources/whatsnew/whats-new-in-v10-2.md

_Generally available in Grafana Enterprise_

With the current release, we enabled RBAC permission validation (`rbac.permission_validation_enabled` setting) by default. This means that the permissions provided in the request during custom role creation or update are validated against the list of [available permissions and their scopes](https://grafana.com/docs/grafana/<GRAFANA VERSION>/administration/roles-and-permissions/access-control/custom-role-actions-scopes/#action-definitions). If the request contains a permission that is not available or the scope of the permission is not valid, the request is rejected with an error message.

## Correlations Editor in Explore

<!-- Kristina Durivage -->

> _Available in public preview in Grafana_

Author: In both cases it's OSS, Enterprise and Cloud. Updated the description, though I'm not sure if we need to explicitly mention Cloud 🤔

> _Available in public preview in Grafana_

Reviewer: I've addressed that in my review and opened a new PR per my review notes.

> Creating correlations has just become easier. Check out our new correlations editor inside Explore by selecting the `+ Add > Add correlation` option from the top bar, or from the command palette. The editor shows all possible places where data links can be placed. The editor will guide you through building and testing target queries. For more details please check [the documentation]({{< relref "../administration/correlations/" >}}).

Author: Sweet! Thanks ❤️  Might be worth updating the template as well, currently it says:

> You must use relative references when linking to docs within the Grafana repo. Please do not use absolute URLs.

> Creating correlations has just become easier. Check out our new correlations editor inside Explore by selecting the `+ Add > Add correlation` option from the top bar, or from the command palette. The editor shows all possible places where data links can be placed. The editor will guide you through building and testing target queries. For more details please check [the documentation]({{< relref "../administration/correlations/" >}}).

Reviewer: Yes, we definitely need to update it! It's on the list of tasks.

> To try out Correlations, enable the `correlations` feature toggle. If you’re using Grafana Cloud and would like to enable this feature, please contact customer support.

Reviewer: ```suggestion
To try out Correlations, enable the `correlations` feature toggle. If you’re using Grafana Cloud and would like to enable this feature, contact Grafana Support.
```

> ## Create correlations for provisioned data sources

Reviewer: ```suggestion
### Create correlations for provisioned data sources
```

<!-- Piotr Jamróz -->

> _Available in public preview in Grafana_

Author: Same as above: 

> In both cases it's OSS, Enterprise and Cloud. Updated the description, though I'm not sure if we need to explicitly mention Cloud 🤔

> You can now create correlations either using administration page or provisioning regardless of whether data source was provisioned or not. In previous versions there was a restriction that if a data source was provisioned, the only way to add correlations to it was also with provisioning. It's no longer the case and you can easily create new correlations mixing both methods - using administration page and/or provisioning.

Reviewer: ```suggestion
You can now create correlations using either the **Administration** page or provisioning, regardless of whether a data source was provisioned or not. In previous versions of Grafana, if a data source was provisioned, the only way to add correlations to it was also with provisioning. Now, that's no longer the case, and you can easily create new correlations mixing both methods—using the **Administration** page or provisioning.
```

> To try out Correlations, enable the `correlations` feature toggle. If you’re using Grafana Cloud and would like to enable this feature, please contact customer support.

Reviewer: ```suggestion
To try out Correlations, enable the `correlations` feature toggle. If you’re using Grafana Cloud and would like to enable this feature, contact Grafana Support.
```

_Generally available in Grafana Enterprise_

With the current release, we enabled RBAC permission validation (`rbac.permission_validation_enabled` setting) by default. This means that the permissions provided in the request during custom role creation or update are validated against the list of [available permissions and their scopes](https://grafana.com/docs/grafana/<GRAFANA VERSION>/administration/roles-and-permissions/access-control/custom-role-actions-scopes/#action-definitions). If the request contains a permission that is not available or the scope of the permission is not valid, the request is rejected with an error message.

## Correlations

### Correlations editor in Explore

<!-- Kristina Durivage -->

> _Available in public preview in Grafana Open Source, Enterprise and Cloud_

Reviewer: Removing mention of the Cloud availability here and I'll add these to the Cloud What's new and ping you for review
```suggestion
_Available in public preview in Grafana Open Source and Enterprise_
```

> Creating correlations has just become easier. Try out our new correlations editor in Explore by selecting the **+ Add > Add correlation** option from the top bar, or from the command palette. The editor shows all possible places where you can place data links and guides you through building and testing target queries. For more information, refer to [the documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/administration/correlations/).

Reviewer: ```suggestion
Creating correlations has just become easier. Try out our new correlations editor in **Explore** by selecting the **+ Add > Add correlation** option from the top bar or from the command palette. The editor shows all possible places where you can place data links and guides you through building and testing target queries. For more information, refer to [the documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/administration/correlations/).
```

> To try out Correlations, enable the `correlations` feature toggle. If you’re using Grafana Cloud and would like to enable this feature, contact Grafana Support.

Reviewer: Sorry still getting used to the separation of on-prem and Cloud release notes. Forgot this sentence doesn't make sense here. It'll be part of the Cloud version of these entries.
```suggestion
To try out Correlations, enable the `correlations` feature toggle.
```

### Create correlations for provisioned data sources

<!-- Piotr Jamróz -->

> _Available in public preview in Grafana Open Source, Enterprise and Cloud_

Reviewer: ```suggestion
_Available in public preview in Grafana Open Source and Enterprise_
```

You can now create correlations using either the **Administration** page or provisioning, regardless of whether a data source was provisioned or not. In previous versions of Grafana, if a data source was provisioned, the only way to add correlations to it was also with provisioning. Now, that's no longer the case, and you can easily create new correlations mixing both methods—using the **Administration** page or provisioning.

> To try out Correlations, enable the `correlations` feature toggle.

Reviewer: ```suggestion
To try out **Correlations**, enable the `correlations` feature toggle.
```

> To try out Correlations, enable the `correlations` feature toggle. If you’re using Grafana Cloud and would like to enable this feature, contact Grafana Support.

Reviewer: ```suggestion
To try out Correlations, enable the `correlations` feature toggle.
```
